### ISSUE TITLE
[Comprehension] User doesn't have enough information to complete a task.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80112]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefits Non-Disability
**Product:** Burial Form 530
**Feature:** 21P-530 Parity Updates

## Findings details
**VA.gov Experience Standard - issue:** User doesn't have enough information to complete a task.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [v2 burial-allowance/statement-of-truth]https://staging.va.gov/burials-and-memorials-v2/application/530/benefits/burial-allowance/statement-of-truth), the text of the statements that you're confirming are true are likely to be missed by screen reader users.

When moving through a form flow like this, screen reader users will typically either be using their software in forms mode or otherwise just be tabbing between interactive elements --- not having their screen reader announce the whole page. The implicit expectation is that any text that's necessary for completing an input will be programmatically associated with that input. But that means helper text that's technically accessible may be missed by screen reader users if it's not associated with any inputs.

If the helper text isn't critical to completing the task, then that's not a huge issue. But if it's important information that the user should really know when answering the questions on the page, we need to ensure that it's properly associated and announced.

## Recommended action

There are a couple of options for resolving this --- totally up to you for what feels right.

**Option 1:** Make all of the helper text part of the `legend` element and then style it so it doesn't look like the `legend` visually, eg.
```
<legend>
   <span class="some-class">Type of burial allowance</span>
   <span class="some-other-class">You selected that...</span>
   ...
</legend>
```

When a screen reader user tabs into the first interactive element in the `fieldset`, everything in the legend will be announced --- so nothing will be missed.

**Option 2:** As I read the text on this page, the most critical parts are the statements themselves:
> - The remains of the Veteran...
> - The Veteran's estate doesn't...

For this one, I think it would be sufficient to include those statements in the `message-aria-describedby` prop on the `va-checkbox` component. Note that these would have to just be formatted as a text string here (no `ul`/`li`). For a screen reader user tabbing to the checkbox, what they would hear would be something like: `Type of burial allowance, grouping. Checkbox, not checked. I confirm these statements are true. The remains of the Veteran have not been claimed by their family members or friends, and the Veteran's estate doesn't have the financial resources to pay for burial and funeral expenses.`

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by steele-lm** [comment]
Sean meeting with Robert to discuss creating new web component for this.

**Comment by Midge-dev** [comment]
No new pattern. Work around was implemented until Platform adds support to their code.
